### PR TITLE
Send newrelic events on login and register

### PR DESCRIPTION
**Pull request by acasajus**

### CODE REVIEW

File: app/auth/views/register.py [hunk 1]
```diff
@@ -60,6 +61,7 @@ def register():
                     hcaptcha_res,
                 )
                 flash("Wrong Captcha", "error")
+                RegisterEvent(RegisterEvent.ActionType.regsiter_catpcha_failed).send()
```
Comment (nguyenkims) [inline app/auth/views/register.py:64]: There seems to be a typo in `regsiter_catpcha_failed`

**Review by nguyenkims — COMMENTED** [review]
Hey I wonder if we should also log these events in api endpoints too?

File: app/auth/views/register.py (hunk 1 above)
Comment (acasajus) [reply app/auth/views/register.py]: Good catch :ok_hand:

**Comment by acasajus** [comment]
> Hey I wonder if we should also log these events in api endpoints too?

Totally forgot about the api routes. I've just added the events there too :+1:

**Review by nguyenkims — APPROVED** [review]
LGTM!

**Pull request merged**